CocoaGrapes/ViewController.swift

```diff
@@ -7,13 +7,55 @@
 //
 
 import UIKit
+import SwiftGifOrigin
+import MarqueeLabel
 
 class ViewController: UIViewController {
 
     @IBOutlet weak var imageView: UIImageView!
+    var label: MarqueeLabel!
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        let dankImage = UIImage.gif(name: "dankgrapes")
+        imageView.image = dankImage
+        
+        label = MarqueeLabel(frame: CGRect.zero, rate: 30, fadeLength: 10.0)
+        label.text = "According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don't care what humans think is impossible."
+        label.animationCurve = .easeInOut
+        label.trailingBuffer = 10
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.isUserInteractionEnabled = true
+        label.holdScrolling = true
+        self.view.addSubview(label)
+        
+        // Tap gesture
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(labelWasTapped))
```

Comment:
There is a way to enable tapping to scroll built into MarqueeLabel, check the documentation